### PR TITLE
set configuration variables for continuous integration

### PR DESCRIPTION
**Pull request by feiloo**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/cio-abcd/variantinterpretation/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### Changes

add a test_ukb.config to enable ci testing on a ukb testserver.
additionally set some needed configuration variables

### CODE REVIEW

**Comment by sci-kai** [comment]
The CI test seems to work great. Before merging, we should add some documentation about the test dataset in the `README.md`.
This should include: Sample, size (no of variants), assay/kit, workflow with version for creating VCF files, type (here: tumor-only sequencing).

File: conf/test_ukb.config
Comment (sci-kai) [inline conf/test_ukb.config:1]: Here you can remove the TODO comments.

**Review by sci-kai — APPROVED** [review]
These look good, if the CI is working it can be rebased and merged with `dev`.

**Review by biolancer — APPROVED** [review]
Needs some cleanup in the Config file as mentioned by sci-kai, but LGTM. Thanks a lot!

**Comment by biolancer** [comment]
After discussion on the 31th august, this PR remains on hold.

**Pull request closed without merging**
